app/helpers/pages_helper.rb

```diff
@@ -217,21 +217,25 @@ def page_object(page)
     base.merge(layouts_and_plugins)
   end
 
-  def countries
-    ISO3166::Country.all.map do |c|
-      {
-        name: c.name,
-        alpha2: c.alpha2,
-        alpha3: c.alpha3,
-        country_code: c.country_code,
-        currency_code: c.currency_code,
-        eu_member: c.in_eu?,
-        eea_member: c.in_eea?
-      }
-    end
-  end
-
   def truncate_page_content(content, length = 500)
     Truncato.truncate(content, max_length: length)
   end
+
+  def plugin_supplemental_data(member, url_params)
+    # TODO
```

Comment:
Perhaps we won't need this, as I'm thinking of creating an API endpoint (to go with a front end function) that will provide the member's supplemental data. That will allow us to cache the plugins config, since we won't have to extend it with member data on every request.

Reply:
👍 to API endpoint, allowing us to extend the view cache.